spec/lucky_cli/runner_spec.cr

```diff
@@ -28,9 +28,13 @@ describe LuckyCli::Runner do
   end
 
   it "does not call the task task is not found" do
+    LuckyCli::Runner.exit_with_error_if_not_found = false
+
     LuckyCli::Runner
       .run(args: ["not.real"])
       .should_not have_called_my_cool_task
+
+    LuckyCli::Runner.exit_with_error_if_not_found = true
```

Comment:
Can you wrap this in a `begin/ensure` so that it reset to `true` even if it raises or fails for some reason?